### PR TITLE
[32384] Marry IFC viewer page and WP list component

### PR DESCRIPTION
**Pull request by HDinger**

### CODE REVIEW

**Comment by HDinger** [comment]
Ticket: 

https://community.openproject.com/projects/bim/work_packages/32384/activity

**Comment by openprojectci** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/opf/openproject/blob/0ead450de2c7282c9c650b77238b819db706dc2d/frontend/src/app/components/modals/wp-destroy-modal/wp-destroy.modal.ts#L44">frontend/src/app/components/modals/wp-destroy-modal/wp-destroy.modal.ts#L44</a> - Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/opf/openproject/blob/0ead450de2c7282c9c650b77238b819db706dc2d/frontend/src/app/components/resizer/wp-resizer.component.ts#L35">frontend/src/app/components/resizer/wp-resizer.component.ts#L35</a> - Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/opf/openproject/blob/0ead450de2c7282c9c650b77238b819db706dc2d/frontend/src/app/components/wp-new/wp-new-split-view.component.ts#L31">frontend/src/app/components/wp-new/wp-new-split-view.component.ts#L31</a> - Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: frontend/src/app/modules/work_packages/routing/wp-full-view/wp-full-view.component.ts [hunk 1]
```diff
@@ -28,17 +28,12 @@
...
 import {StateService} from '@uirouter/core';
 import {TypeResource} from 'core-app/modules/hal/resources/type-resource';
 import {Component, Injector, OnInit} from '@angular/core';
 import {WorkPackageViewSelectionService} from 'core-app/modules/work_packages/routing/wp-view-base/view-services/wp-view-selection.service';
-import {States} from 'core-components/states.service';
-import {KeepTabService} from 'core-components/wp-single-view-tabs/keep-tab/keep-tab.service';
-import {FirstRouteService} from "core-app/modules/router/first-route-service";
 import {WorkPackageSingleViewBase} from "core-app/modules/work_packages/routing/wp-view-base/work-package-single-view.base";
 import {BackRoutingService} from "core-app/modules/common/back-routing/back-routing.service";
-import {InjectField} from "core-app/helpers/angular/inject-field.decorator";
```
Comment (openprojectci) [inline frontend/src/app/modules/work_packages/routing/wp-full-view/wp-full-view.component.ts:37]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/0ead450de2c7282c9c650b77238b819db706dc2d/frontend/src/app/modules/work_packages/routing/wp-full-view/wp-full-view.component.ts#L37"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>

File: frontend/src/app/modules/ifc_models/bcf/container/bcf-container.component.ts [hunk 2]
```diff
@@ -13,9 +13,6 @@ import {untilComponentDestroyed} from "ng2-rx-componentdestroyed";
```
Comment (openprojectci) [inline frontend/src/app/modules/ifc_models/bcf/container/bcf-container.component.ts:13]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/0ead450de2c7282c9c650b77238b819db706dc2d/frontend/src/app/modules/ifc_models/bcf/container/bcf-container.component.ts#L13"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>

**Pull request merged**